main.py

```diff
@@ -1,7 +1,35 @@
-import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
 
+import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genre_list = ["Western", "Action", "Dramma"]
+    actor_list = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for genre in genre_list:
+        Genre.objects.create(name=genre)
+
+    for actor in actor_list:
+        Actor.objects.create(first_name=actor[0], last_name=actor[1])
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/224999946-deb2a90d-b5dc-4590-ba36-6c1b7da0d4f9.png)
What if it will be another Klooney in db? u have to update George according to the task